rebase_test.go

```diff
@@ -315,14 +444,40 @@ func commitSomething(repo *Repository, something, content string) (*Oid, error)
 	}
 	defer newTree.Free()
 
-	if err != nil {
-		return nil, err
-	}
```

Comment:
can this block be kept?

Reply:
It's already checked directly above, I don't think it needs to do it again?